android/assets/jsons/translations/template.properties

```diff
@@ -710,6 +710,7 @@ HIGHLY EXPERIMENTAL - YOU HAVE BEEN WARNED! =
 Enable portrait orientation = 
 Generate translation files = 
 Translation files are generated successfully. = 
+Fastlane files are generated successfully. =
```

Comment:
This line needs to end with a space after =

Reply:
Thank you!